Blish HUD/GameServices/ArcDps/Models/Ev.cs

```diff
@@ -42,63 +57,78 @@ public class Ev {
         /// </summary>
+        /// <remarks>
+        /// For strips and cleanses: <seealso cref="SrcAgent">SrcAgent</seealso> = relevant, <seealso cref="DstAgent">DstAgent</seealso> = caused it.
+        /// </remarks>
         public bool   IsBuffRemove    { get; }
         /// <summary>
-        /// TRUE if SrcAgent is above 90% health. Otherwise FALSE.
+        /// TRUE if <seealso cref="SrcAgent">SrcAgent</seealso> is above 90% health. Otherwise FALSE.
         /// </summary>
         public bool   IsNinety        { get; }
         /// <summary>
-        /// TRUE if DstAgent is below 50% health. Otherwise FALSE.
+        /// TRUE if <seealso cref="DstAgent">DstAgent</seealso> is below 50% health. Otherwise FALSE.
         /// </summary>
         public bool   IsFifty         { get; }
         /// <summary>
-        /// TRUE if SrcAgent is moving at time of event. Otherwise FALSE.
+        /// TRUE if <seealso cref="SrcAgent">SrcAgent</seealso> is moving at time of event. Otherwise FALSE.
         /// </summary>
         public bool   IsMoving        { get; }
         /// <summary>
-        /// TRUE if a state change occured (SrcAgent is now alive, dead, downed and other ambigious stuff ex. when SrcAgent is self, DstAgent is a reward id and Value is a reward type such as a wiggly box.). Otherwise FALSE.
+        /// TRUE if a state change occured. Otherwise FALSE.
         /// </summary>
+        /// <remarks>
+        /// <seealso cref="SrcAgent">SrcAgent</seealso> is now alive, dead, downed and other ambiguous stuff eg. when <seealso cref="SrcAgent">SrcAgent</seealso> is <seealso cref="Ag.Self">Self</seealso>, <seealso cref="DstAgent">DstAgent</seealso> is a reward id and <seealso cref="Value">Value</seealso> is a reward type such as a wiggly box.
+        /// </remarks>
         public bool   IsStateChange   { get; }
         /// <summary>
-        /// TRUE if SrcAgent is flanking DstAgent at time of event. Otherwise FALSE.
+        /// TRUE if <seealso cref="SrcAgent">SrcAgent</seealso> is flanking <seealso cref="DstAgent">DstAgent</seealso> at time of event. Otherwise FALSE.
         /// </summary>
         public bool   IsFlanking      { get; }
         /// <summary>
-        /// TRUE if all or part of damage was VS. barrier/shield. Otherwise FALSE.
+        /// TRUE if all or part of damage was VS. barrier or shield. Otherwise FALSE.
         /// </summary>
         public bool   IsShields       { get; }
         /// <summary>
-        /// FALSE if buff damage happened during tick. Otherwise TRUE.
+        /// TRUE if no buff damage happened during tick. Otherwise FALSE.
         /// </summary>
         public bool   IsOffCycle      { get; }
         /// <summary>
-        /// Buff instance id of buff applied. Zero if buff damage happened during tick, non-zero otherwise.
+        /// Buff instance id of buff applied. Non-zero if no buff damage happened during tick. Otherwise zero.
```

Comment:
no buff damage during tick sounds a little confusing to me.
What do you think about `TRUE if buff event did not happen as part of the tick` or something along the lines, not really sure tbh.

Reply:
Yeah it's because of the double negation. I thought it sounded weird but it's the change you wanted.
I am not sure if ``TRUE if buff event did not happen as part of the tick`` is the same as ``no buff damage happening`` because a buff can cause 0 damage and be relevant.

Reply:
This is the only thing left but no clue how to phrase it better. I do not find your suggestion particular convincing either, because the phrase from the official doc states "buff damage" specifically.